jme3-core/src/main/java/com/jme3/renderer/Caps.java

```diff
@@ -396,6 +397,15 @@
      */
     BinaryShader;
 
+    private static final Caps[] VALUES = values();
```

Comment:
Why this? What's the use case?
Why not just let the user do enum.values()[i] and handle the bounds check?

Reply:
enum.values() recalculetes for each call.

Reply:
Ok... but what's the use case?

Reply:
@Nehon I have answered as a comment in the changes.